src/rust/ide/span-tree/src/generate.rs

```diff
@@ -232,10 +261,16 @@ fn prefix_can_be_chained_with_parent(ctx:ChainingContext) -> bool {
 mod test {
     use super::*;
 
+    use crate::builder::TreeBuilder;
+    use crate::node::Kind::*;
+
     use wasm_bindgen_test::wasm_bindgen_test;
     use parser::Parser;
-    use crate::builder::{TreeBuilder, Builder};
-    use ast::crumbs::{InfixCrumb, PrefixCrumb, SectionLeftCrumb, SectionRightCrumb, SectionSidesCrumb};
+    use ast::crumbs::InfixCrumb;
+    use ast::crumbs::PrefixCrumb;
+    use ast::crumbs::SectionLeftCrumb;
+    use ast::crumbs::SectionRightCrumb;
+    use ast::crumbs::SectionSidesCrumb;
```

Comment:
Not in order. 

IIRC I've added `wasm_bindgen_test::wasm_bindgen_test` to the prelude for test configuration, so it should be possible to safely remove it.

Reply:
I'm surprised, that I got no warning about redundant imports.